requirements/pytorch/strategies.txt

```diff
@@ -1,5 +1,5 @@
 fairscale>=0.4.5, <=0.4.6
-deepspeed<0.6.0
+deepspeed>=0.6.0, <0.6.5
```

Comment:
For the released package, it remains unpinned if I recall correctly, so this should be good. 
The limitation is only in Lite afaik.